src/components/Match/MatchSummary.tsx

```diff
@@ -0,0 +1,55 @@
+import { Fragment } from 'react';
+import { MainReport, MatchSummary } from '../../types/common';
+import FormattedNumber from '../common/FormattedNumber';
+import MatchResultComponent from '../common/MatchResult';
+import MatchUserSummaryComponent from './MatchUserSummary';
+
+interface Props {
+    match: MatchSummary;
+    previousMatch?: MatchSummary;
+    userIDs: string[];
+    report: MainReport;
+}
+
+export default function MatchSummaryComponent({ match, userIDs, report, previousMatch }: Props) {
```

Comment:
`*Component` part is maybe a bit redundant?

Reply:
The "Component" suffix prevents collisions with interfaces/types with the same name.
Alternatively, we could use the "I" prefix for all interfaces e.g. "IMatchSummary", but I don't really like this convention.

Reply:
See https://github.com/adrianstanescu/lol-stats/issues/3